dev/jenkins/Dockerfile-jdk11

```diff
@@ -105,7 +105,7 @@ RUN mkdir -p /home/jenkins && \
     chmod -R 777 /.config && \
     apt-get update -y && \
     apt-get upgrade -y ca-certificates && \
-    apt-get install -y build-essential fuse libfuse-dev make ruby ruby-dev
+    apt-get install -y build-essential fuse libfuse-dev libfuse3-3 libfuse3-dev make ruby ruby-dev
```

Comment:
why for fuse 2 : fuse, libfuse-dev are installed
but for fuse 3: libfuse3-3 libfuse3-dev are installed

can fuse library works for both libfuse 2 & 3
why we don't need to install libfuse package?

Reply:
Thanks for spotting this. I didn't notice package `fuse3` exists and thought `libfuse3-3` is the counterpart of `fuse`. It should be replaced by `fuse3`. `fuse3` contains `libfuse3-3` and a few other basic packages.

I'm assuming you are talking about fuse kernel module. Although some functionalities require higher kernel version, from my testing AlluxioFuse works without any problems.
Both `fuse` and `fuse3` contain libfuse packages.